### PR TITLE
feat(timeout): Added timeouts for steps

### PR DESCRIPTION
**Pull request by Vorobeyko**
## Motivation/Description of the PR
- Description of this PR, which problem it solves
- Resolves #issueId (if applicable).

Applicable helpers:

- [ ] WebDriver
- [ ] Puppeteer
- [ ] Nightmare
- [ ] REST
- [ ] FileHelper
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe
- [ ] Playwright

Applicable plugins:

- [ ] allure
- [ ] autoDelay
- [ ] autoLogin
- [ ] customLocator
- [ ] pauseOnFail
- [ ] puppeteerCoverage
- [ ] retryFailedStep
- [ ] screenshotOnFail
- [ ] selenoid
- [ ] stepByStepReport
- [ ] wdio

## Type of change

- [ ] :fire: Breaking changes
- [ ] :rocket: New functionality
- [ ] :bug: Bug fix
- [ ] :clipboard: Documentation changes/updates
- [ ] :hotsprings: Hot fix
- [ ] :hammer: Markdown files fix - not related to source code
- [ ] :nail_care: Polish code

## Checklist:

- [ ] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [ ] Lint checking (Run `npm run lint`)
- [ ] Local tests are passed (Run `npm test`)

### CODE REVIEW

File: lib/actor.js [hunk 1]
```diff
@@ -116,7 +117,16 @@ function recordStep(step, args) {
       event.emit(event.step.started, step);
       step.startTime = Date.now();
     }
-    return val = step.run(...args);
+    console.log(step);
```
Comment (DavertMik) [inline lib/actor.js:120]:
```suggestion
```

File: lib/config.js [hunk 2]
```diff
@@ -83,7 +84,7 @@ class Config {
   /**
    * Get current config.
    * @param {string} key
-   * @param {*} val
+   * @param {*=} val
```
Comment (DavertMik) [inline lib/config.js:87]: ??

File: test/runner/steps_test.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+const path = require('path');
+const exec = require('child_process').exec;
+
+const runner = path.join(__dirname, '/../../bin/codecept.js');
+const codecept_dir = path.join(__dirname, '/../data/sandbox/configs/steps');
+const codecept_run = `${runner} run`;
+const config_run_config = (config, grep) => `${codecept_run} --config ${codecept_dir}/${config} ${grep ? `--grep "${grep}"` : ''}`;
+
+describe('CodeceptJS Steps', () => {
+  it('should stop test, when step exceeded', (done) => {
```
Comment (PeterNgTr) [inline test/runner/steps_test.js:11]: I"m just curious if those tests can ben written in async/await approach?

Comment (Vorobeyko) [reply test/runner/steps_test.js:11]: yes, it can be.

File: lib/config.js (hunk 2 above)
Comment (Vorobeyko) [reply lib/config.js:87]: This changes for typings. Because this param is an optional

**Pull request merged**